chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,437 @@
-                    ],
-                    "path": [
-                        "rest",
-                        "certified_products",
-                        "10457"
-                    ]
-                },
-                "description": "Expect Listing Basic response to have svapResultsUrl"
-            },
-            "response": []
-        }
-    ],
-    "protocolProfileBehavior": {}
+	"info": {
+		"_postman_id": "8b7e235b-2704-42b7-b3ed-42fef144e70c",
+		"name": "certified-product-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET : certified_product: data driven tests",
+			"item": [
+				{
+					"name": "GET /certified_products- 2015 listings",
+					"item": [
+						{
+							"name": "GET /{certifiedProductId}/measures - return status 200",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											"var certifiedProductId= pm.iterationData.get(\"certifiedProductId\");\r",
+											"pm.environment.set(\"certifiedProductId\",\"certifiedProductId\");"
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"{certifiedProductId}/measures return 200 status for valid API key\", function () {\r",
+											"    pm.response.to.have.status(200);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"type": "text",
+										"value": "{{apiKey}}"
+									},
+									{
+										"key": "Content-Type",
+										"type": "text",
+										"value": "application/json"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/{{certifiedProductId}}/measures",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"{{certifiedProductId}}",
+										"measures"
+									]
+								},
+								"description": "{certifiedProductId}/measures return 200 status for valid API key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{certifiedProductId} - returns all RWT fields",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											"var certifiedProductId= pm.iterationData.get(\"certifiedProductId\");\r",
+											"pm.environment.set(\"certifiedProductId\",\"certifiedProductId\");"
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"var expresult = pm.response.json()\r",
+											"pm.test(\"/{certifiedProductId} end point response should have all RWT fields\", function () {\r",
```

Comment:
I think we should always add a test about status code of the request.

Reply:
Added